docs/content/Examples-Tutorials-Recipes/Building-context-aware-pre-aggregations.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Building Context-Aware Pre-aggregations
+permalink: /recipes/building-context-aware-pre-aggregations
+category: Examples & Tutorials
+subCategory: Query acceleration
+menuOrder: 6
+---
+
+## Use case
+
+Using different environments to test a product is good practice. Let's imagine
+we have three environments: testing, staging, and production. We want to use
+multiple database schemas depending on the current environment. In addition, we
+want to use pre-aggregation for each schema. So, in the recipe below, we'll
+learn how dynamically select database schema and create a pre-aggregation with
+the scheduled update.
+
+## Data schema
+
+To select the database schema, we will use the
+[`COMPILE_CONTEXT`](https://cube.dev/docs/schema/reference/cube#context-variables-compile-context)
+global variable. Let's pass into it an `env` variable from
+[`securityContext`](https://cube.dev/docs/security/context) with our environment
+value.
```

Comment:
The sentence starting with 'Let's pass into it...' is a bit confusing

Reply:
Changed

Reply:
Much better 😀